### ISSUE TITLE
Nuckalavees: too strong?

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#72826]
### Is your feature request related to a problem? Please describe.

These things are cracked. They have 200 speed and hit as hard as a hulk and spam poison gas and the knockback smash. Unironically, if you see one, you are probably dead. They can commonly spawn in labs due to the tear-in-reality vault.

### Solution you would like.

The issue is that nucks are both incredibly fast and deadly. The other "beefy" nether mob, the flying polyp, has even better stats, but uses the hit-and-run behavior which makes it much more fair as an enemy. The nuck just glues to you and then annihilates you basically instantly. Thus some reduction to their speed, frequency of their smash attack, reduction in the poison gas they produce, and/or giving them the hit-and-run behavior might make them more reasonable.

### Describe alternatives you have considered.

Make them highly exclusive, such as only in the monster corpse? They're too strong for a common spawn IMO.

**Comment by fairyarmadillo** [comment]
There are portals all over the world map now which has led to several anecdotal incidents of people walking around, seeing some kind of a poison gas cloud in a field, wandering over to investigate and getting instantly bodied by a 200 speed superboss.

They also cause a lot of lag #71717

**Comment by TheSaddestGoomba** [comment]
I don't think they need stat adjustments so much as spawn adjustments. They die to a rifle pretty easily, I've killed two with an AR15 in my current run. Granted those were both on open ground with good LoS.
Melee sounds like a nightmare against them and indoor spawns make that much more likely.
If it's possible to make them an outdoor-only spawn I think that would be a better first step.

**Comment by I-am-Erk** [comment]
If they are as powerful as reported in melee, then they can't be spawning out on the street at random. it doesn't necessarily matter if there's counterplay, it's still the kind of 'gotcha' we're trying to avoid. "You see a monster in a cloud of gas on day two, you consider attacking with your fire ax, you instantly die" is not good gameplay.

Putting them in monster corpses might be a good alternative, or having them be something whose arrival is triggered by something else. As long as there is agency, such as "I went inside this obviously scary and dangerous place" or "I did that thing they said would destabilize the portals" it's fine if they're super tough.
